src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheckTest.java

```diff
@@ -83,6 +83,20 @@ public void testCountEmptyIsFalse() throws Exception {
                 getPath("InputMethodLengthCountEmptyIsFalse.java"), expected);
     }
 
+    @Test
+    public void testCountEmptyIsFalsetwo() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWithInlineConfigParser(
+                getPath("InputMethodLengthCountEmptyIsFalsetwo.java"), expected);
```

Comment:
```suggestion
        verifyWithInlineConfigParser(
                getPath("InputMethodLengthCountEmptyIsFalsetwo.java"), CommonUtil.EMPTY_STRING_ARRAY);
```

Reply:
Sorry,  keep imperative.
It is not imperative, but still keep as is